[2022-04-06 00:47:37,478] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-06 00:47:37,704] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2014-06-01T00:00:00+00:00 [queued]>
[2022-04-06 00:47:37,705] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-06 00:47:37,705] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-06 00:47:37,705] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-06 00:47:37,827] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): processing_tasks_posts_answers.ingest_posts_answers_to_bucket> on 2014-06-01 00:00:00+00:00
[2022-04-06 00:47:37,964] {standard_task_runner.py:52} INFO - Started process 106267 to run task
[2022-04-06 00:47:37,999] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'IngestToGCP', 'processing_tasks_posts_answers.ingest_posts_answers_to_bucket', 'scheduled__2014-06-01T00:00:00+00:00', '--job-id', '10055', '--raw', '--subdir', 'DAGS_FOLDER/ingest_dag.py', '--cfg-path', '/tmp/tmp5l5wpprm', '--error-file', '/tmp/tmpp1ftlfmq']
[2022-04-06 00:47:38,000] {standard_task_runner.py:80} INFO - Job 10055: Subtask processing_tasks_posts_answers.ingest_posts_answers_to_bucket
[2022-04-06 00:47:38,858] {logging_mixin.py:109} INFO - Running <TaskInstance: IngestToGCP.processing_tasks_posts_answers.ingest_posts_answers_to_bucket scheduled__2014-06-01T00:00:00+00:00 [running]> on host 9fe4286b2855
[2022-04-06 00:47:39,639] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-06 00:47:39,775] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=IngestToGCP
AIRFLOW_CTX_TASK_ID=processing_tasks_posts_answers.ingest_posts_answers_to_bucket
AIRFLOW_CTX_EXECUTION_DATE=2014-06-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-06-01T00:00:00+00:00
[2022-04-06 00:47:42,905] {python.py:175} INFO - Done. Returned value was: None
[2022-04-06 00:47:42,984] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=IngestToGCP, task_id=processing_tasks_posts_answers.ingest_posts_answers_to_bucket, execution_date=20140601T000000, start_date=20220406T004737, end_date=20220406T004742
[2022-04-06 00:47:43,140] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-04-06 00:47:43,146] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 106267. PIDs of all processes in the group: [106267]
[2022-04-06 00:47:43,146] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 106267
[2022-04-06 00:47:43,274] {process_utils.py:70} INFO - Process psutil.Process(pid=106267, status='terminated', exitcode=0, started='00:47:37') (106267) terminated with exit code 0
